numpy/f2py/rules.py

```diff
@@ -202,7 +199,7 @@
 \tif (PyErr_Occurred())
 \t\t{PyErr_SetString(PyExc_ImportError, \"can't initialize module #modulename# (failed to import numpy)\"); return m;}
 \td = PyModule_GetDict(m);
-\ts = PyUnicode_FromString(\"$R""" + """evision: $\");
+\ts = PyUnicode_FromString(\"#f2py_version#\");
```

Comment:
Seems in `from scipy.linalg.blas import _fblas` the version actually used to be just `_fblas.__version__ == b"$Revision: $"`?

In any case, this looks good to me. Should we add a release note? It might seem like we are decrementing the version from `2` to `1.20`?

A fewline below, we are setting:
```
__f2py_numpy_version__
```
which IIRC is also new, should we still do that or really just switch over entirely?

Reply:
No need for a release note I think.